tutorials/from_darknet.py

```diff
@@ -0,0 +1,216 @@
+def save_lib():
+    '''Save the graph, params and .so to the current directory'''
+    print("Saving the compiled output...")
+    path_name = 'nnvm_darknet_' + model_name
+    path_lib = path_name + '_deploy_lib.so'
+    lib.export_library(path_lib)
+    with open(path_name 
++ "deploy_graph.json", "w") as fo:
+        fo.write(graph.json())
+    with open(path_name 
++ "deploy_param.params", "wb") as fo:
+        fo.write(nnvm.compiler.save_param_dict(params))
+#save_lib()
+
+######################################################################
+# Load a test image
+# --------------------------------------------------------------------
+print("Loading the test image...")
+test_image = 'dog.jpg'
+img_url = 'https://github.com/siju-samuel/darknet/blob/master/data/' + \
+            test_image   +'?raw=true'
+download(img_url, test_image)
+
+data = nnvm.frontend.darknet.load_image(test_image, net.w, net.h)
+######################################################################
+# Execute on TVM
+# --------------------------------------------------------------------
+# The process is no different from other examples.
+from tvm.contrib import graph_runtime
+ctx = tvm.cpu(0)
+
+m = graph_runtime.create(graph, lib, ctx)
+
+# set inputs
+m.set_input('data', tvm.nd.array(data.astype(dtype)))
+m.set_input(**params)
+# execute
+print("Predicing the test image...")
+
+start_time = time.time()
+m.run()
+timediff = (time.time() - start_time)
+# get outputs
+out_shape = (net.outputs,)
+tvm_out = m.get_output(0, tvm.nd.empty(out_shape, dtype)).asnumpy()
+top1 = np.argmax(tvm_out)
+print("TVM Run Time = %s seconds." % timediff)
+print('TVM Prediction output id : ', top1)
+
+#####################################################################
+# Look up synset name
+# --------------------------------------------------------------------
+# Look up prdiction top 1 index in 1000 class imagenet.
+out_name_file = {}
+out_name_file['yolo'] = "coco.names"
+out_name_file['yolo-voc'] = "voc.names"
+out_name_file['alexnet'] = "imagenet.shortnames.list"
+out_name_file['resnet50'] = "imagenet.shortnames.list"
+out_name_file['resnet152'] = "imagenet.shortnames.list"
+out_name_file['extraction'] = "imagenet.shortnames.list"
+
+imagenet_name = out_name_file[model_name]
+imagenet_url = 'https://github.com/siju-samuel/darknet/blob/master/data/' \
+                + imagenet_name +'?raw=true'
+
+download(imagenet_url, imagenet_name)
+with open(imagenet_name) as f:
+    imagenet = f.readlines()
+
+print("TVM Predicted result : ", imagenet[top1])
+
+#####################################################################
+# confirm correctness with darknet output
+# --------------------------------------------------------------------
+start_time = time.time()
+darknet_lib.network_predict_image(net, darknet_lib.load_image_color(test_image.encode('utf-8'), 0, 0))
+print("DARKNET Run Time = %s seconds." % (time.time() - start_time))
```

Comment:
as tutorial, the final step is not necessary, we could however, show things like bounding boxes in the final image